lib/logic/api/helper.js

```diff
@@ -32,31 +46,9 @@ const sendHttpRequest = async (httpOptions, authContext, throwOnUnauthorized, ti
     debug('Sending http request:\n%O', finalOptions);
     let response;
     try {
-        response = await rp(finalOptions);
+        response = await request(finalOptions);
     } catch (err) {
-        debug('Response:\n%O', err.response.body);
-
-        if (_.isEqual(err.statusCode, 401)) {
-            const error = new CFError({
-                cause: err,
-                message: 'Error: Please create or update your authentication context',
-            });
-
-            if (!throwOnUnauthorized) {
-                printError(error);
-                process.exit(1);
-            } else {
-                throw error;
-            }
-        }
-        if (_.isEqual(err.statusCode, 403)) {
-            printError(new CFError({
-                cause: err,
-                message: 'Error: You do not have permissions to perform this action',
-            }));
-            process.exit(1);
-        }
-
+        debug('Response:\n%O', response);
```

Comment:
this part that handles the error will never be called with the new request library, and thus this should be moved correctly downwards.